gateways/node_gateway.py

```diff
@@ -1,8 +1,9 @@
-from typing import Callable, List, Union
+from typing import Callable, Dict, List, Union
 
 from common.configuration import DATA_AGGREGATOR_LAMBDA_NAME, NODE_CACHE_TTL
```

Comment:
I know that this was not made in this PR, but I saw we are configuring the name of the Lambda using this Env Var `DATA_AGGREGATOR_LAMBDA_NAME`.

I think it should be better to just configure it directly in serverless.yml, I think we can make it inject the name of the lambda in the env var there.

Reply:
created an issue for this #80